index.js

```diff
@@ -0,0 +1,29 @@
+; (() => {
+    const appElement = document.getElementById("app")
+
+    const fetchData = () => {
+        // Fetch data from API
+
+        const loader = document.createElement("div")
+        loader.className = "loader"
+
+        appElement.appendChild(loader)
+
+        // const settings = {
+        // 	async: true,
+        // 	crossDomain: true,
+        // 	url: 'https://cars-by-api-ninjas.p.rapidapi.com/v1/cars?model=corolla',
+        // 	method: 'GET',
+        // 	headers: {
+        // 		'X-RapidAPI-Key': 'SIGN-UP-FOR-KEY',
```

Comment:
Ok, here is a bit thing YOU CANNOT COMMIT YOUR KEY NEVER

here is how to go solve it:
1. you create enviromental variable and put your key there
2. in your code you use this environmental variable instead your key

Reply:
But this will be in after PR as to show spinner you dont see to fetch anything, so I am not sure why this commented code is here